### ISSUE TITLE
Device linked with "addDevice" can send but not receive (Dbus)

### ISSUE THREAD

**Issue by bublath** [AsamK/signal-cli#993]
I was just trying out to link two signal-cli instances together by using the link/addDevice.
With "link" I created the sgnl://linkdevice?uuid=.... URI on the unregistered instance (which is actually a multi-account instance that already has other numbers registered) and fed this URI into the addDevice call on the "master" instance.
This worked without problem, and I could find the new number in the "unregistered" device and could activate it (start using the new Dbus object).
Sending messages from this new instance works, but replies only go to the "master".
It also seems unable to retrieve the list of groups, while getting the contact list works.

Am I doing anything wrong or is this a bug?

**Comment by AsamK** [comment]
Receiving messages should work. If the contact list was synced that means that at least the sync messages have already been received by the newly linked device.
Groups aren't synced yet (until #604 is implemented completely), they will only appear on the linked device, once a message from a specific group is received.

**Comment by bublath** [comment]
I went into a bit more analysis and compared the difference between linking a mobile phone with the app (signal-cli generating the qrcode) and linking a signal-cli instance using the "addDevice". I was simply using "dbus-monitor --system" to make sure to catch everything.
When I link my mobile phone I get standard MessageReceived (+V2) and everything works.
When the device was added using "addDevice" I'm getting SyncMessageReceived (+V2) instead. Content pretty much looks correct (meaning as would the body of MessageReceived) but it's the wrong signal, that's why I'm not recognizing it.
